src/flist/flist_case_studies/flist_nextcloud_case_study.md

```diff
@@ -0,0 +1,544 @@
+<h1> FList Case Study: Nextcloud All-in-One </h1>
+
+![nextcloud_logo](./img/nextcloud_logo.jpeg)
+
+<h2> Table of Contents </h2>
+
+- [Introduction](#introduction)
+  - [FList: What is It?](#flist-what-is-it)
+  - [Case Study Objective](#case-study-objective)
+  - [The Overall Process](#the-overall-process)
+- [Docker Image Creation](#docker-image-creation)
+  - [Nextcloud FList Directory Tree](#nextcloud-flist-directory-tree)
+  - [Dockerfile](#dockerfile)
+  - [README.md File](#readmemd-file)
+  - [scripts Folder](#scripts-folder)
+    - [sshd\_init.sh](#sshd_initsh)
```

Comment:
why `\_` not `/` ?

Reply:
The title uses the filename with an underscore sshd_init

\_ is to get the literal symbol. The interactive TOC works well with this.

Reply:
@xmonader 
Please let me know if you have any questions concerning this.